### PR TITLE
Two Color Adjustment Filters Added

### PR DESCRIPTION
**Pull request by tapos-datta**
* ColorMatrix
* ColorMonochrome

### CODE REVIEW

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package com.linkedin.android.litr.filter.video.gl;
+
+import android.graphics.PointF;
+import android.opengl.GLES20;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Created by tapos-datta on 9/14/20.
```
Comment (izzytwosheds) [inline litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java:9]: Small request: can you add BSD licensing header in the beginning (other classes have it) and move this line to the end of it?

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+import android.graphics.PointF;
+import android.opengl.GLES20;
+
+import androidx.annotation.NonNull;
+
+import java.nio.FloatBuffer;
+
+/**
+ * Created by tapos-datta on 9/14/20.
```
Comment (izzytwosheds) [inline litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java:11]: Same here

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        super(DEFAULT_VERTEX_SHADER, FRAGMENT_SHADER, size, position, rotation);
+
+        this.intensity = intensity;
+        this.inputColorRGB = inputColorRGB;
+    }
+
+    @Override
+    protected void applyCustomGlAttributes() {
+        GLES20.glUniform1f(getHandle("intensity"), intensity);
+        GLES20.glUniform3fv(getHandle("newColor"),1, FloatBuffer.wrap(inputColorRGB));
```
Comment (izzytwosheds) [inline litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java:73]: nip: space before `1`

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+    private static final String FRAGMENT_SHADER =
+                    "#extension GL_OES_EGL_image_external : require\n" +
+
+                    "precision mediump float;\n" +
+                    "varying highp vec2 vTextureCoord;\n" +
+                    "uniform lowp samplerExternalOES sTexture;" +
+                    "uniform lowp vec3 newColor;" +
+                    "uniform lowp float intensity;\n" +
+                    "\n"+
```
Comment (izzytwosheds) [inline litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java:23]: nip: can you match shader styling with other shaders? ("{" on a separate line, no "\n" lines, identation outside of shader text)

**Review by izzytwosheds — COMMENTED** [review]
Excellent work! Just few styling fixes and we should be good to go.

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java (hunk 1 above)
Comment (izzytwosheds) [reply litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java:9]: Actually, if this is coming from `android-gpuimage` project, we should use their copyright notice

Comment (tapos-datta) [reply litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java]: Sure, I have updated with corresponding licenses into the source files.

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java (hunk 3 above)
Comment (tapos-datta) [reply litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java]: Refactored sources.

File: litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java (hunk 4 above)
Comment (tapos-datta) [reply litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java]: Matched shader styling with other shaders.

**Pull request merged**